README.md

```diff
@@ -18,9 +18,9 @@ The following guidelines should be taken into consideration before submitting an
 
 - Once a forum discussion has started, a link to the discussion should be shared publicly in the #governance channel of the Regen Network [Discord Server](https://discord.gg/regen-network) and on Telegram and Twitter (tagging [@regen_network](https://twitter.com/regen_network)).
 
-- The time between socializing the proposal and submitting the proposal will vary depending on the proposal and community engagement but generally a proposal should be brought to discussion *at least* one week before submitting the proposal on chain.
+- The time between socializing the proposal and submitting the proposal will vary depending on the proposal and community engagement but generally a proposal should be brought to discussion on commonwealth *at least* two weeks before submitting the proposal on chain.
```

Comment:
We may want to have an official proposal for this change. The idea with proposal 10 was to update the voting period but to also introduce guidelines that are updated through a governance process:

> With that being said, any significant changes to the guidelines should go through the governance process in the form of a text proposal.

I think this change along with any others would be the result of [Formalize the Use of Commonwealth](https://forum.regen.network/discussion/4328-proposal-for-discussion-formalize-the-use-of-commonwealth).

Reply:
```suggestion
- The time between socializing the proposal and submitting the proposal will vary depending on the proposal and community engagement but generally a proposal should be brought to discussion on [Commonwealth](https://forum.regen.network/) *at least* two weeks before submitting the proposal on chain.
```